### DISCUSSION TITLE
¿Donde puedo encontrar los iconos para la práctica de css?

### DISCUSSION THREAD

**Discussion by AlexAntonioG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#379] [Q&A] [answered]
Que tal, me preguntaba acerca de donde puedo encontrar los iconos que se utilizan en la pagina de css, ya que los he estado buscando y no los encuentro, en un momento había visto que alguien ya habló del tema pero no encuentro la discusión 😢 

Agradecería mucho si me apoyan , gracias 😸

**Comment by strdbr** [comment, marked as answer]
https://fontawesome.com

Acá están casi todos, igual te da el tutorial de cómo meterlos directo al HTML al loggearte

**Comment by OscarPedroza** [comment]
También puedes obtener de este sitio: [https://fontastic.me/](https://fontastic.me/)
